### ISSUE TITLE
[BUG] Base API Url for OpenRouter causes duplicate URL part

### ISSUE THREAD

**Issue by tobitege** [jeffdapaz/VisualChatGPTStudio#74]
Installed the extension in Visual Studio 2019 as of today.
In VS extension options I specified "https://openrouter.ai/api" as "Base API URL" (tried with and without "/v1" at the end!).
The Request URI is then malformed with a duplicate "v1/" in it:
https://openrouter.ai/api/v1/v1/chat/completions

**Comment by jeffdapaz** [comment]
Hi @tobitege,

I will verify and say something here when I have an update.

**Comment by jeffdapaz** [comment]
@tobitege,

I already published a new release with the fix.

Give a try and say something if was not fixed. 

Only remember that you do not need include the version on "Base URL" parameter.

**Comment by tobitege** [comment]
Hey Jeff,
thank you so much for the fix!
Just for fyi, I had to set up some values differently to make it work for OpenRouter:
"Stop Sequences" must apparently be filled so I put "stop" in it.
"Base API URL" set to "https://openrouter.ai/api"
"Model Language Override" has to conform to OpenRouter' syntax like "google/gemini-flash-1.5"

I then used the Solution Context for my app and selected some C# files from it and I was able to ask about them and got explanations as reply, as I hoped. :)

Thanks again, for me it seems to work now!
